test/Microsoft.ML.Functional.Tests/Datasets/TrivialMatrixFactorization.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Functional.Tests.Datasets
+{
+    /// <summary>
+    /// A class containing one property per <see cref="DataKind"/>.
```

Comment:
>/// A class containing one property per <see cref="DataKind"/>. [](start = 4, length = 63)

I don't get this sentence. What DataKind has to do with MF? Your data is row+col+value triplets for matrix.

Reply:
Copy/Paste error, unfortunately.

---
In reply to: [259511254](https://github.com/dotnet/machinelearning/pull/2646#discussion_r259511254) [](ancestors = 259511254)